.travis.yml

```diff
@@ -7,16 +7,16 @@ env:
     - IPFS_REUSEPORT=false
```

Comment:
Does this `IPFS_REUSEPORT=false` even do anything?  Why is this here?

Reply:
Yes. It disables reuseport. It's probably best to leave it there (again, this was probably there for testing on osx but reuseport is finicky so I wouldn't break *these* tests just to suss out errors in reuseport).

Reply:
But where is reuseport called?